crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -303,6 +311,11 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for TrustpayPaymentsRequest {
             ip_address: browser_info.ip_address,
         };
         let params = get_mandatory_fields(item)?;
+        let billing_last_name = item
+            .get_billing()?
```

Comment:
We do not want the payment to proceed if billing details are not passed, is that okay?

Reply:
yes, billing details are mandatory for trustpay
https://github.com/juspay/hyperswitch/blob/32c222786430c50ffac6f73e0fed72276da9e1d2/crates/router/src/connector/trustpay/transformers.rs#L200